CacheAdvance.podspec

```diff
@@ -1,15 +1,15 @@
 Pod::Spec.new do |s|
   s.name     = 'CacheAdvance'
-  s.version  = '2.0.0'
+  s.version  = '2.1.0'
   s.license  = 'Apache License, Version 2.0'
   s.summary  = 'A performant cache for logging systems. CacheAdvance persists log events 30x faster than SQLite.'
   s.homepage = 'https://github.com/dfed/CacheAdvance'
   s.authors  = 'Dan Federman'
   s.source   = { :git => 'https://github.com/dfed/CacheAdvance.git', :tag => s.version }
-  s.swift_version = '5.6'
```

Comment:
We compile on Swift 5.6, 5.7, 5.8, and 5.9. To validate that we work on visionOS, we have to lint on Swift 5.9, which we can't do if our podspec specifies an earlier source.